### DISCUSSION TITLE
Not-prose conditional modifier

### DISCUSSION THREAD

**Discussion by thedevdavid** [tailwindlabs/tailwindcss#12082] [Help] [answered]
Hey there!

I was wondering, if I have an element, let's keep it simple `<h2 className="mb-0">FAQ</h2>`

I want to reuse this element both in and out of a `prose` parent. And I only want to apply styles if it's NOT in a `prose` parent. Is there a way to conditionally apply `not-prose` classes, like `<h2 className="not-prose:text-2xl not-prose:font-bold mb-0">FAQ</h2>`.

Thanks a lot!

**Comment by wongjn** [comment, marked as answer]
You could use an arbitrary variant like `[&:not(.prose_*)]:`. You could then abstract this as a named variant via a Tailwind plugin: https://play.tailwindcss.com/PHTj7ocEMM

**Reply by thedevdavid** [reply]
That makes sense. Thank you!
